exp-models/addon/models/experiment.js

```diff
@@ -19,9 +20,9 @@ export default DS.Model.extend(JamModel, {
 
     title: DS.attr('string'),
     description: DS.attr('string'),
-    beginDate: DS.attr('date'),	// TODO: ISODate
-    endDate: DS.attr('date'),	// TODO: ISODate
-    lastEdited: DS.attr('date'),	// TODO: ISODate
+    beginDate: DS.attr('date'), // TODO: ISODate
```

Comment:
These TODO lines are unnecessary- modern versions of ember use ISO date format already without the need for further transforms. 

(I could have sworn I removed those comments before, but I guess not?)